### ISSUE TITLE
Disable indexing the folder links

### ISSUE THREAD

**Issue by zliangfeng** [Microsoft/vscode#59287]
Issue Type: <b>Feature Request</b>

I build a folder link in my project, then `coder helper` try to indexing, and my app is freezing. I quit code, delete the link, it goes well. Is there any way to disable folder links indexing?

VS Code version: Code 1.27.2 (f46c4c469d6e6d8c46f268d1553c5dc4b475840f, 2018-09-12T07:04:11.329Z)
OS version: Darwin x64 17.7.0

**Comment by bpasero** [comment]
@zliangfeng can you find out the full arguments of the code helper that does this? You can e.g. use the process explorer for this (Help menu).

**Comment by zliangfeng** [comment]
@bpasero here is my project structure:
.
|-- my-project
|   |-- node_modules
|   |-- smb_share -> /

I created a soft link `smb_share` to root, because a node package needs a path as part of arguments. It seems the code helper indexing all files below root (eh, I created the link by mistake).

**Comment by bpasero** [comment]
Can you follow this wiki and post the results back here to understand more about why performance is so bad for you: https://github.com/Microsoft/vscode/wiki/Performance-Issues

Thanks!

**Comment by zliangfeng** [comment]
I tried to reproduce the issue with Extensions Disabled, 
In the terminal, run: ln -s / smb_share
snapshot below:
![image](https://user-images.githubusercontent.com/9211566/46122623-4d00c800-c24c-11e8-92bf-0e6f98de9ead.png)
In the meantime, the OS warned me VS Code were trying to access photo library.

Then I deleted the soft link, and restarted VS Code, the problem went away!  
![image](https://user-images.githubusercontent.com/9211566/46122695-cc8e9700-c24c-11e8-8faa-659fb17ca6b4.png)  

It's ok, I'll be careful to create soft link in projects.
